tutorials/week1-installing-postman.md

```diff
@@ -0,0 +1,186 @@
+    ```
+
+- **Accept**: Specifies the format in which the client expects the server to return the response (e.g., `application/json`).
+  - *Example*:
+    ```
+    Accept: application/json
+    ```
+
+### 5. **Response**
+After the server processes the request, it sends back a **response**. The response typically includes:
+
+- **Status Code**: A number indicating the result of the request.
+  - Common status codes:
+    - `200 OK`: The request was successful.
+    - `201 Created`: The resource was successfully created (used for POST requests).
+    - `400 Bad Request`: There was an error in the request.
+    - `404 Not Found`: The resource could not be found.
+    - `500 Internal Server Error`: An error occurred on the server.
+
+- **Response Body**: Contains the data returned by the API, usually in JSON format. For example, when creating a new user, the server might return a response like this:
+  ```json
+  {
+      "id": 1,
+      "username": "newuser",
+      "email": "newuser@example.com"
+  }
+  ```
+
+Mozilla documentation for reference: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status
+
+## Steps to Install Postman
+
+### Step 1: Download Postman
+1. Open your web browser and navigate to the official Postman website: [https://www.postman.com/downloads/](https://www.postman.com/downloads/).
+2. Select the version that is compatible with your operating system (Windows/macOS). If you're using a Mac, be sure to download the version that corresponds to your system's chip (Intel or Apple).
+3. Click the "Download" button.
+
+### Step 2: Install Postman
+
+#### For Windows:
+1. Once the download is complete, find the setup file in your downloads folder (typically named `Postman-win64-setup.exe`).
+2. Double-click the file to start the installation.
+3. Follow the on-screen instructions to complete the installation. Postman will install automatically and open once the process is finished.
+
+#### For macOS:
+1. Open the downloaded `.dmg` file.
+2. Drag the Postman app icon into the "Applications" folder.
+3. Once copied, go to your Applications folder and double-click the Postman icon to launch the app.
+
+### Step 3: Create an Account (Optional but Recommended)
+Once Postman is installed:
+1. Open the Postman application.
+2. You will be prompted to sign in or create a Postman account.
+
+## Using Postman: Basic Steps
+
+### Step 0: Ensure your backend is running
+
+
+### Step 1: Launch Postman
+After installation, launch Postman. You'll see a workspace where you can start creating requests.
+
+### Step 2: Make Your First API Request
+1. Click on the "New" button at the top-left and select **HTTP**.
+
+![Postman1](./assets/week1-installing-postman/Postman1.png)
+
+2. In the request window, choose the type of request method (GET, POST, PUT, DELETE) from the dropdown menu (default is GET).
+3. Enter the URL of the API you want to test in the input field (e.g., `https://localhost:8000/api/users`).
```

Comment:
We should update this example URL to match the screenshot example for consistency

Reply:
Updated this to match the postman image